victims/settings.py

```diff
@@ -10,6 +10,8 @@
 HOST = config("HOST", default="0.0.0.0")
 PORT = config("PORT", default="8000", cast=int)
 STATIC_DIR = Path() / "victims" / "static"
+# 557 chars + ... = 560 chars
+CASE_MAX_CHARS = config("CASE_MAX_CHARS", default=557, cast=int)
```

Comment:
Can we use the ellipsis character (`…`) instead of three periods (`...`)? If so this number changes to 559 ; )

Reply:
done